docs/src/lib/utils.md

```diff
@@ -9,4 +9,7 @@ end
 
 ```@docs
 sign_cadlag
+@neutral
+@absorbing
+@neutral_and_absorbing
```

Comment:
is `@neutral_and_absorbing` a remainder?

Reply:
oops, fixed